src/reflect/scala/reflect/internal/Names.scala

```diff
@@ -80,7 +80,7 @@ trait Names extends api.Names {
     while (i < len) {
       if (nc + i == _chrs.length) {
         val newchrs = new Array[Char](_chrs.length * 2)
-        java.lang.System.arraycopy(_chrs, 0, newchrs, 0, chrs.length)
+        java.lang.System.arraycopy(_chrs, 0, newchrs, 0, _chrs.length)
```

Comment:
Someone was just counting, and we should add, "Off by underscore."

One could omit `java.lang` here, without compromising oneself.

Ha, `Don't access name table contents directly!` Irony of high order.

It should be, `chrs is deprecated, did you mean _chrs?`

Reply:
There's a comment on `private[this] var _chrs`, `// TODO this ought to be private`, which speaks to the churn all this namesy code was undergoing earlier this year. By inlining `chrs`, I hope to save a few tenths of a nanosecond per compilation, until it gets inlined, you can thank me later.

Is `_chrs.length` accessing the contents directly, or just the table?